docs/user/dashboard/tsvb.asciidoc

```diff
@@ -53,36 +31,66 @@ When you use only {data-sources}, you are able to:
+
+. On the dashboard, click *All types*, then select *TSVB*.
+
+. In *TSVB*, click *Panel options*, then specify the *Data* settings.
 
-. Click *Panel options*, then open the *Index pattern selection mode* options next to the *Index pattern* dropdown.
+. Open the *Index pattern selection mode* options next to the *Index pattern* dropdown.
 
-. Select *Use only Kibana index patterns*.
+. Select *Use only Kibana {data-sources}*.
 
-. From the *Index pattern* drodpown, select the index pattern, then select the *Time field*.
+. From the *Index pattern* drodpown, select the {data-source}, then select the *Time field* and *Interval*.
+
+. Select a *Drop last bucket* option. 
 +
-image::images/tsvb_index_pattern_selection_mode.png[Change index pattern selection mode action]
+By default, *TSVB* drops the last bucket because the time filter intersects the time range of the last bucket. To view the partial data, select *No*.
+
+. To view a filtered set of documents, enter <<kuery-query, KQL filters>> in the *Panel filter* field.
 
 [float]
-[[configure-the-data-series]]
+[[tsvb-function-reference]]
 ==== Configure the series
 
 Each *TSVB* visualization shares the same options to create a *Series*. Each series can be thought of as a separate {es} aggregation. 
-For each series, the *Options* control the styling and {es} options, and are inherited from *Panel options*.
+The *Options* control the styling and {es} options, and are inherited from *Panel options*.
 When you have separate options for each series, you can compare different {es} indices, and view two time ranges from the same index. 
 
 To configure the value of each series, select the function, then configure the function inputs. Only the last function is displayed.
 
-. From the *Aggregation* dropdown, select the function for the series. 
+. From the *Aggregation* dropdown, select the function for the series. *TSVB* provides you with shortcuts for some frequently-used functions:
++
+*Filter Ratio*::
+  Returns a percent value by calculating a metric on two sets of documents. 
+  For example, calculate the error rate as a percentage of the overall events over time.
++
+*Counter Rate*::
+  Used when dealing with monotonically increasing counters. Shortcut for *Max*, *Derivative*, and *Positive Only*.
++
+*Positive Only*::
+  Removes any negative values from the results, which can be used as a post-processing step
+  after a derivative.
++
+*Series Agg*::
+  Applies a function to all of the *Group by* series to reduce the values to a single number.
+  This function must always be the last metric in the series.
+  For example, if the *Time Series* visualization shows 10 series, the sum *Series Agg* calculates
+  the sum of all 10 bars and output a single Y value per X value. This is often confused
+  with the overall sum function, which outputs a single Y value per unique series.
++
+*Math*::
+  The math context is able to do simple and advanced calculations per series.
```

Comment:
math context > math function?

Reply:
Ugh. I don't even know. I didn't write this. I'll zhush it up!